sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -669,3 +669,7 @@ CREATE TABLE `api_token` (
 ) ENGINE = InnoDB;
 #EndIf
 
+#IfMissingColumn users patient_menu_role
```

Comment:
above line will not work :)

Reply:
I did not add the api_token table creation lines.